packages/box/box.ts

```diff
@@ -205,14 +210,14 @@ const Box = {
         tempDirPath,
         destination,
         boxConfig,
-        unpackBoxOptions,
+        unpackBoxOptions
       );
 
       events.emit("unbox:cleaningTempFiles:start");
       tempDirCleanup();
       events.emit("unbox:cleaningTempFiles:succeed");
 
-      utils.setUpBox(boxConfig, destination, events);
+      await utils.setUpBox(boxConfig, destination, events, setUpBoxOptions);
```

Comment:
Only real change in this file, everything else is husky lint. The function is now async because inquirer needs to wait for answers.